packages/auth/lib/modular/index.d.ts

```diff
@@ -642,7 +672,7 @@ export function updateProfile(
 export function verifyBeforeUpdateEmail(
   user: FirebaseAuthTypes.User,
   newEmail: string,
-  actionCodeSettings?: FirebaseAuthTypes.ActionCodeSettings,
+  actionCodeSettings?: FirebaseAuthTypes.ActionCodeSettings | null,
```

Comment:
It does seem a touch redundant to have the type as `actionCodeSettings?` and also have `| null`. but firebase-js-sdk has it, so I guess it stays